app.rb

```diff
@@ -0,0 +1,19 @@
+require "sinatra"
```

Comment:
It's really nice that you have your interface combined in 1 view. This makes the app user friendly.